docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trellix-mvisio-epo-source.md

```diff
@@ -0,0 +1,122 @@
+---
+id: trellix-mvisio-epo-source
+title: Trellix mVision ePO Source
+sidebar_label: Trellix mVision ePO
+description: Learn how to collect event logs using the Trellix mVision ePO.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/trellix-logo.png')} alt="trellix-logo" width="70" />
+
+Trellix is a cybersecurity company that provides cloud-based security solutions for cybersecurity attacks. It provides hardware, software, and services to investigate cybersecurity attacks, protects against malicious software, and analyzes IT security risks.
+
+mVision ePO is a key component of the Trellix security management platform, which provides unified management of endpoint, network, and data security. This can reduce incident response time, strengthen protection, simplify and automate risk and security management, and provide end-to-end network visibility and security.
+
+## Data Source
+
+The Trellix integration fetches event logs using the [Trellix mVision ePO API](https://developer.manage.trellix.com/mvision/apis/v2-events).
+
+## Set up and configuration
+
+The Trellix mVision ePO source requires you to provide a Client ID, Client Secret, and API key. To obtain these, follow the steps below.
+
+1. Log in to the [Trellix Developer Portal](https://developer.manage.trellix.com/).
+1. Go to the **Self service** menu and click [**API Access Management**](https://developer.manage.trellix.com/mvision/selfservice/access_manag).<br/><img src={useBaseUrl('img/send-data/self-service.png')} alt="self-service.png" style={{border: '1px solid black'}} width="600" />
```

Comment:
```suggestion
1. Go to the **Self service** menu, and click [**API Access Management**](https://developer.manage.trellix.com/mvision/selfservice/access_manag).<br/><img src={useBaseUrl('img/send-data/self-service.png')} alt="self-service.png" style={{border: '1px solid black'}} width="600" />
```

Reply:
Go to the **Self service** menu, and click [**API Access Management**] tab. 

Can you write the action button name there? I think it's a tab, check plz.

Reply:
**API Access Management** is an option in Self service menu. So I will call it as option.